loglist/diff_check.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loglist
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"reflect"
+  "sort"
+)
+
+// warningCollector is an interface for structs that have the capability to
+// collect warnings.  It is used to report warnings from the various check
+// functions defined in this file.
+type warningCollector interface {
+	addWarning(string)
+}
+
+type warningList struct {
+	warnings []string
+}
+
+func (wl *warningList) addWarning(w string) {
+	if w != "" {
+		wl.warnings = append(wl.warnings, w)
+	}
+}
+
+// Check: operators lists are identical.
```

Comment:
Can a branched log list include logs from entirely new operators?

Reply:
Looks like it can. Updated the logic.